src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__active"></div>
```

Comment:
<img width="893" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/5b07d0a9-0845-419d-9784-fcb5c421b726">
We should have the ability to change the number of active stars just by changing the modifier

Reply:
Still not fixed from the previous review, you need to remove extra class